processors/blockprocessor/sample_test.go

```diff
@@ -0,0 +1 @@
+package blockprocessor
```

Comment:
What's the purpose of this file?

Reply:
Lets drop the per-plugin sample tests. [This general test gets us most of the way there](https://github.com/algorand/indexer/pull/1315/files#diff-03dff77f7636e05b34cf95cbb0ecd02024030d89eb953e260c937418c05ee69c).